integrations/FacebookPixel/browser.js

```diff
@@ -499,7 +499,8 @@ class FacebookPixel {
   }
 
   getContentType(rudderElement, defaultValue) {
-    const { options, properties } = rudderElement.message;
+    const { properties } = rudderElement.message;
+    const options = rudderElement?.message?.integrations?.FACEBOOK_PIXEL;
```

Comment:
we don't need the key `options`, instead just the `contentType` from rudderElement?.message?.integrations?.FACEBOOK_PIXEL

Reply:
Yes absolutely - we can simplify the whole function quite a bit here actually.

Reply:
I've updated this all and tried to do some manual testing for the various use cases. Let me know what you think.